src/js/views/views/HistoryAll.js

```diff
@@ -44,12 +54,40 @@ const HistoryAll = ({seasons = []}) => {
             countText = null;
         }
 
+        /*
+         * Special cases for team names.
+         */
+
+        // Champ
+        //
+        // Wrap it in a SPAN.
+        let champEl = <span>{teamName(s.champ, s.season)}{countText}</span>;
+
+        // "Champ" is our team.
+        if (s.champ && s.champ.tid === g.userTid) {
+            champEl = {
+                classNames: 'info',
+                data: champEl,
```

Comment:
Good idea, I was thinking of something similar :)

One suggestion... change `data` to `value`, I like how it sounds better, especially since this is the child of another property named data and `row.data.data` is weird.